cmd/boulder-ca/main.go

```diff
@@ -32,6 +32,7 @@ func main() {
 
 		paDbMap, err := sa.NewDbMap(c.PA.DBConnect)
 		cmd.FailOnError(err, "Couldn't connect to policy database")
+
 		pa, err := policy.NewPolicyAuthorityImpl(paDbMap, c.PA.EnforcePolicyWhitelist, c.PA.Challenges)
```

Comment:
Nevermind, didn't notice it moved to the constructor, meaning [above](https://github.com/letsencrypt/boulder/pull/1146/files#diff-c3df615576f96379a790fb1e67e6aec1R136) needs to be removed instead...